R/map.R

```diff
@@ -153,15 +153,18 @@ map_chr <- function(.x, .f, ..., .progress = FALSE) {
 map_ <- function(.type, .x, .f, ..., .progress = FALSE, ..error_call = caller_env()) {
   .x <- vctrs_vec_compat(.x)
   vec_assert(.x, arg = ".x", call = ..error_call)
+  n <- vec_size(.x)
```

Comment:
Isn't it better to be consistent with the naming convention within the function? (IMO it's also a bit misleading to call it size, because it is more like length due to `vctrs_vec_compat`)